### ISSUE TITLE
xhtml2pdf==0.2.13 fails to build pycairo

### ISSUE THREAD

**Issue by stiangrim** [xhtml2pdf/xhtml2pdf#745]
### Describe the Bug

Upgrading from version 0.2.11 --> 0.2.13 gives me a building error in Google Cloud Run.

### Additional Information

```
Step #0 - "build image":   Building wheel for pycairo (pyproject.toml): started
Step #0 - "build image":   Building wheel for pycairo (pyproject.toml): finished with status 'error'
Step #0 - "build image":   error: subprocess-exited-with-error
Step #0 - "build image":   
Step #0 - "build image":   × Building wheel for pycairo (pyproject.toml) did not run successfully.
Step #0 - "build image":   │ exit code: 1
Step #0 - "build image":   ╰─> [12 lines of output]
Step #0 - "build image":       running bdist_wheel
Step #0 - "build image":       running build
Step #0 - "build image":       running build_py
Step #0 - "build image":       creating build
Step #0 - "build image":       creating build/lib.linux-x86_64-cpython-311
Step #0 - "build image":       creating build/lib.linux-x86_64-cpython-311/cairo
Step #0 - "build image":       copying cairo/__init__.py -> build/lib.linux-x86_64-cpython-311/cairo
Step #0 - "build image":       copying cairo/__init__.pyi -> build/lib.linux-x86_64-cpython-311/cairo
Step #0 - "build image":       copying cairo/py.typed -> build/lib.linux-x86_64-cpython-311/cairo
Step #0 - "build image":       running build_ext
Step #0 - "build image":       'pkg-config' not found.
Step #0 - "build image":       Command ['pkg-config', '--print-errors', '--exists', 'cairo >= 1.15.10']
Step #0 - "build image":       [end of output]
Step #0 - "build image":   
Step #0 - "build image":   note: This error originates from a subprocess, and is likely not a problem with pip.
Step #0 - "build image":   ERROR: Failed building wheel for pycairo
Step #0 - "build image": Failed to build pycairo
Step #0 - "build image": ERROR: Could not build wheels for pycairo, which is required to install pyproject.toml-based projects
```

### System Information

OS version: python:3.11-slim-bookworm
Python version: python:3.11
XHTML2PDF version: 0.2.13

**Comment by stefan6419846** [comment]
This is not a *xhtml2pdf* issue, but related to the *reportlab* installation which currently links against *pycairo*. See #741 as well.

**Comment by timobrembeck** [comment]
Fixed in #742

**Comment by timobrembeck** [comment]
@stiangrim I released version `0.2.14` now, does this fix the issue for you?
As noted in the release notes, you might have to install `xhtml2pdf[renderpm]` to make use of the fallback rendering backend instead.

**Comment by stiangrim** [comment]
Thanks @timobrembeck 
It seems to work now!
